data-raw/nuts_data/nuts_de.R

```diff
@@ -0,0 +1,14 @@
+# NUTS3 to zip file
+# Source from European Commission: https://gisco-services.ec.europa.eu/tercet/flat-files
+nuts_de <- read_csv2(
```

Comment:
I think `read_csv2()` is only necessary because .csv files in EU are sometimes separated by ";" not ",". Right? But I think it's confusing to call "Comma separated file (csv)" to a "semi-colon separated file". Instead I would read it with `read_csv2()` interactively, then write it with `write_csv()` and store that version. That way in data-raw/ we can read it more naturally with `read_csv()`. 

Alternatively you can also write with `write_tsv()` and read with `read_tsv()` or `vroom::vroom()`.